src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -122,7 +122,10 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     executionResult match {
       case SuccessfulExecution(outputs, returnCode) => context.parent ! WorkflowActor.CallCompleted(key, outputs, returnCode)
       case AbortedExecution => context.parent ! WorkflowActor.AbortComplete(key)
-      case FailedExecution(e, returnCode) => context.parent ! WorkflowActor.CallFailed(key, returnCode, e.getMessage)
+      case FailedExecution(e, returnCode) =>
+        logger.error("Failing call: " + e.getMessage, e)
+        log.error(e, e.getMessage)
```

Comment:
Was thinking about this and here are my two objections, both of which are easily resolvable:

Semantics: it seems really weird that server logging isn't default for server mode. Perhaps change it to file?

Verbosity: perhaps this is no longer true but at one point console mode logged less stuff. I'd rather not have a crippled logger be used as default.

(Also highlighting, but that's my own personal oddity - sometimes I spin up server just as a shortcut to avoid the highlighting)